src/screens/Settings/CustomerMapping/EditCustomerMappingComponent.jsx

```diff
@@ -418,30 +418,33 @@ export default function EditCustomerMappingComponentBackup({ match }) {
     });
   };
 
-  const handleRatioInput = (index) => (e) => {
+  const handleRatioInput = (index) => async (e) => {
     e.preventDefault();
-    const newValue = parseInt(e.target.value) || 0;
+    const value = parseInt(e?.target?.value) || 0;
 
-    if (newValue > 100) {
+    if (value > 100) {
       e.target.value = 0;
+      ratiowiseData[index] = 0;
       alert("Cannot Enter More than 100 !!!");
     } else {
-      const newData = [...userData];
-      newData[index] = { user_id: userDropdown[index]?.value, ratio: newValue };
```

Comment:
createCustomerMapping file : 
handleGetDepartmentUsers function :
Why 2 filters are used?

Reply:
update code in one filter